recipes/openmm-ml/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openmm-ml" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/openmm/openmm-ml.git
+  git_rev: 5b577c2586232d3ac6f41de064d3290619b2a8aa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - openmm >=7.7 # TODO https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build once merged into a feedstock, depend on dev for now
```

Comment:
> We are in the middle of coordinating the simultaneous release of a few packages so I would appreciate a temporary relaxation of the policy to use git_url Once we get it in a staged recipe and get everything lined up, we will switch to using tarballs for the release.

I think if you add the `conda_build_config.yaml` with channel targets to this PR, this package will be published to your pre-release label. Is there a reason you can't do all the coordination under a pre-release label?

Reply:
@carterbox we can! I wasn't aware I could use that option before the staged recipe was migrated to a feedstock.